### ISSUE TITLE
Saint Croix National Scenic Riverway label

### ISSUE THREAD

**Issue by chadlawlis** [nationalparkservice/places-data#141]
It appears Saint Croix is labeling from a centroid, which is placing the label relatively far off the river. 

![saint_croix](https://cloud.githubusercontent.com/assets/1238636/9891037/ac89af12-5bc0-11e5-8846-bd1faa5ab74a.jpg)

It should be aligned with the line.

**Comment by chadlawlis** [comment]
http://insidemaps.nps.gov/park-tiles-3/#8/45.510/-92.017

cc @mtaylorlong. Let me know if you would like the primary label point moved or other label-only records added to address this.

**Comment by mtaylorlong** [comment]
I moved the primary label point SE so that its placed inside the actual polygon. While I was in the area, I also alligned the min_zoom settings for Mississippi, Saint Croix, and Lower Saint Croix so they would appear more consistent amongst eachother. I moved the MISS label slightly to the East so it would fall within its polygon as well.

I'm assuming this one closed unless @cwlawlis802 wants to open it back up for any reason.
